deluge/ui/web/server.py

```diff
@@ -668,9 +695,10 @@ def start_ssl(self):
             certificate = Certificate.loadPEM(cert.read()).original
         with open(configmanager.get_config_dir(self.pkey)) as pkey:
             private_key = KeyPair.load(pkey.read(), FILETYPE_PEM).original
-        options = CertificateOptions(privateKey=private_key, certificate=certificate, method=SSL.SSLv23_METHOD)
+        ciphers = AcceptableCiphers.fromOpenSSLCipherString(TLS_CIPHERS)
+        options = CertificateOptions(privateKey=private_key, certificate=certificate,
+                                     insecurelyLowerMinimumTo=TLSVersion.TLSv1_2, acceptableCiphers=ciphers)
```

Comment:
I don't think we want `insecurelyLowerMinimumTo`, see https://twistedmatrix.com/documents/current/core/howto/ssl.html#tls-protocol-options

I have set TLS 1.2 for Python 3 changes using: `raiseMinimumTo=TLSVersion.TLSv1_2`

Do you think that the minimum that Twisted require will be an issue?

Reply:
No, I don't think so.
BTW, your change forced me to merge the two branches manually.